Source/Urho3D/Core/Thread.cpp

```diff
@@ -41,7 +41,7 @@
 namespace Urho3D
 {
 
-#ifdef URHO3D_THREADING
+#ifdef URHO3D_THREADING_GENERAL
```

Comment:
im not thrilled about `URHO3D_THREADING_GENERAL` vs `URHO3D_THREADING_WORKERS` outside of WorkQueue...